pkg/descriptions.json

```diff
@@ -1997,18 +1997,11 @@
     "Name": "System.Text.Json",
     "Description": "Provides a fast, non-cached, forward-only way to read UTF-8 encoded JavaScript Object Notation (JSON) text.",
     "CommonTypes": [
+      "System.Text.Json.JsonDocument",
       "System.Text.Json.Utf8JsonReader",
       "System.Text.Json.Utf8JsonWriter",
-      "System.Text.Json.JsonDocument"
-    ]
-  },
-  {
-    "Name": "Microsoft.Bcl.Json.Sources",
-    "Description": "Provides the source code implementation of System.Text.Json as internal types. This package is only required for projects targeting .NET Framework 4.6.1+ or .NET Core 2.x; and is not for projects that can target .NET Core 3.0 and higher. For guidance on consuming the source pacakge, visit: https://aka.ms/json-source-package-guide.",
-    "CommonTypes": [
-      "System.Text.Json.Utf8JsonReader",
-      "System.Text.Json.Utf8JsonWriter",
-      "System.Text.Json.JsonDocument"
+      "System.Text.Json.JsonElement",
+      "System.Text.Json.Serialization.JsonSerializer"
```

Comment:
It's no longer in the Serialization namespace, is it?

Reply:
mm it does seem like it is:

https://github.com/dotnet/corefx/blob/1c1b78f7ec1d04aeac059d3d4e4c975d3a3ad23e/src/System.Text.Json/ref/System.Text.Json.cs#L361-L405

Reply:
Look at master.

Reply:
@stephentoub's correct. The `JsonSerializer` is in the S.T.Json namespace now.